src/actions/src/index.ts

```diff
@@ -22,13 +22,21 @@
 export * from './actions';
 
 // kepler.gl actions accessible outside component
+export * from './action-wrapper';
 export * from './vis-state-actions';
 export * from './ui-state-actions';
 export * from './map-state-actions';
 export * from './map-style-actions';
 export * from './identity-actions';
 export * from './provider-actions';
 
+/* eslint-disable prettier/prettier */
```

Comment:
why do we disable prettier here?

Reply:
That's because it throws an exception for every `export type {...` line. If you  have an idea how it can be reconfigured - let me know, I'll change the eslint config